src/SMSSpamCollection/SMSSpamCollection.jl

```diff
@@ -0,0 +1,138 @@
+
+
+3. ABOUT
+--------
+
+The corpus has been collected by Tiago Agostinho de Almeida (http://www.dt.fee.unicamp.br/~tiago) and José María Gómez Hidalgo (http://www.esp.uem.es/jmgomez).
+
+We would like to thank Dr. Min-Yen Kan (http://www.comp.nus.edu.sg/~kanmy/) and his team for making the NUS SMS Corpus available. See: http://www.comp.nus.edu.sg/~rpnlpir/downloads/corpora/smsCorpus/. He is currently collecting a bigger SMS corpus at: http://wing.comp.nus.edu.sg:8080/SMSCorpus/
+
+4. LICENSE/DISCLAIMER
+---------------------
+
+We would appreciate if:
+
+- In case you find this corpus useful, please make a reference to previous paper and the web page: http://www.dt.fee.unicamp.br/~tiago/smsspamcollection/ in your papers, research, etc.
+- Send us a message to tiago@dt.fee.unicamp.br in case you make use of the corpus.
+
+The SMS Spam Collection v.1 is provided for free and with no limitations excepting:
+
+1. Tiago Agostinho de Almeida and José María Gómez Hidalgo hold the copyrigth (c) for the SMS Spam Collection v.1.
+
+2. No Warranty/Use At Your Risk. THE CORPUS IS MADE AT NO CHARGE. ACCORDINGLY, THE CORPUS IS PROVIDED `AS IS,' WITHOUT WARRANTY OF ANY KIND, INCLUDING WITHOUT LIMITATION THE WARRANTIES THAT THEY ARE MERCHANTABLE, FIT FOR A PARTICULAR PURPOSE OR NON-INFRINGING. YOU ARE SOLELY RESPONSIBLE FOR YOUR USE, DISTRIBUTION, MODIFICATION, REPRODUCTION AND PUBLICATION OF THE CORPUS AND ANY DERIVATIVE WORKS THEREOF BY YOU AND ANY OF YOUR SUBLICENSEES (COLLECTIVELY, `YOUR CORPUS USE'). THE ENTIRE RISK AS TO YOUR CORPUS USE IS BORNE BY YOU. YOU AGREE TO INDEMNIFY AND HOLD THE COPYRIGHT HOLDERS, AND THEIR AFFILIATES HARMLESS FROM ANY CLAIMS ARISING FROM OR RELATING TO YOUR CORPUS USE.
+
+3. Limitation of Liability. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR THEIR AFFILIATES, OR THE CORPUS CONTRIBUTING EDITORS, BE LIABLE FOR ANY INDIRECT, SPECIAL, INCIDENTAL OR CONSEQUENTIAL DAMAGES, INCLUDING, WITHOUT LIMITATION, DAMAGES FOR LOSS OF GOODWILL OR ANY AND ALL OTHER COMMERCIAL DAMAGES OR LOSSES, EVEN IF ADVISED OF THE POSSIBILITY THEREOF, AND REGARDLESS OF WHETHER ANY CLAIM IS BASED UPON ANY CONTRACT, TORT OR OTHER LEGAL OR EQUITABLE THEORY, RELATING OR ARISING FROM THE CORPUS, YOUR CORPUS USE OR THIS LICENSE AGREEMENT.
+
+# Interface
+- [`SMSSpamCollection.features`](@ref)
+- [`SMSSpamCollection.targets`](@ref)
+
+"""
+
+module SMSSpamCollection
+
+using DataDeps
+using DelimitedFiles
+
+export features, targets
+
+const DATA = joinpath(@__DIR__, "spam.csv")
+
+"""
+    targets(; dir = nothing)
+
+Get the targets for the SMS Spam Collection dataset,
+a 5574 element array listing the targets for each sample.
+
+```jldoctest
+julia> using MLDatasets: SMSSpamCollection
+
+julia> targets = SMSSpamCollection.targets()
+
+julia> summary(targets)
+"1×5574 Matrix{Any}"
+
+julia> targets[1]
+"ham"
+```
+"""
+function targets(; dir = nothing)
```

Comment:
why you provide a `dir` argument and then don't use it?

Reply:
Um, I removed it now.